packages/autoprofile/package.json

```diff
@@ -33,7 +33,7 @@
     "detect-libc": "^1.0.3",
     "nan": "^2.14.2",
     "node-abi": "^2.30.0",
-    "node-gyp": "^7.1.2"
+    "node-gyp": "^8.2.0"
```

Comment:
I'm not 100% sure if we want to do this because node-gyp 8 removes support for Python 2 and only supports Python 3 going forward. Of course, Python 2 is EOL since over 18 months or so but I could imagine that some systems out there are still on Python 2.

Is the move to node-gyp 8 strictly necessary to get rid of the audit warning for node-tar? If so, we can consider it. Since `@instana/autoprofile` is an optional dependency of `@instana/collector`, it should not break anything anyway.

Reply:
Wasn't aware of python 2. That's defo too breaky. Something for a new major.
I'll push a different fix in some mins